server/controllers/authController/index.js

```diff
@@ -30,19 +30,23 @@ const AuthController = {
   },
   update: async (req, res) => {
     try {
-      const { password, confirmpw, reset_key, username } = req.body
+      const { password, confirmPassword, reset_key, username } = req.body
 
-      if (password !== confirmpw)
+      if (password !== confirmPassword)
         return res.status(400).json({ error: 'passwords do not match' })
 
       const { appDb } = req.app.locals
       const encryptPassword = hash(password)
       const user = await UserModel.findOne(appDb, {
         uid: username,
         reset_key,
+        force_reset_pw: true,
```

Comment:
So after the user resets their password this will prompt them to reset their password? I'm definitely missing something.

Reply:
It's a weird flow... an admin makes a reset password hash and when it is submitted it also adds this property and sets it to true, after it's updated it's set to false.